doc/cmd/completion/script.md

```diff
@@ -0,0 +1,57 @@
+# Driftctl completion script
```

Comment:
We should add a link to this page from the main user guide documentation `doc/README.md`

Reply:
Added a completion line with a link to the doc in 814eb1a